src/tmx/TmxUtils/src/simulation/SimulationUtils.cpp

```diff
@@ -0,0 +1,30 @@
+#include <simulation/SimulationEnvVar.h>
+
+namespace tmx::utils::sim{
+    bool is_simulation_mode() {
```

Comment:
Please pull develop and rebase on develop. This file has been renamed to SimulationEnvUtils. As the PR currently stands this file will be added again under your PR.